modules/item.rb

```diff
@@ -0,0 +1,37 @@
+class Item
+  attr_accessor :genre, :author, :published_date
+  attr_reader :id, :archived, :label
+
+  def initialize(published_date, archived: true)
+    @id = Random.rand(0...1000)
+    @published_date = published_date
+    @archived = archived
+  end
+
+  def move_to_archive
+    @archived = true if can_be_archived?
+  end
+
+  # def label=(label)
+  #   @label = label
+  #   @label.items.push(self) unless label.items.include?(self)
+  # end
```

Comment:
- [ ] it would be better to uncomment these lines of code in order to achieve one to many relationships and also please make sure to add :label as attr_reader